nifi-nar-bundles/nifi-stateless-processor-bundle/nifi-stateless-processor/src/main/java/org/apache/nifi/processors/stateless/ExecuteStateless.java

```diff
@@ -850,6 +862,8 @@ private StatelessEngineConfiguration createEngineConfiguration(final ProcessCont
             contentRepoDirectory = null;
         }
 
+        final String statusTaskSchedule = context.getProperty(STATUS_TASK_SCHEDULE).isSet() ? context.getProperty(STATUS_TASK_SCHEDULE).getValue() : null;
```

Comment:
Can just use `final String statusTaskSchedule = context.getProperty(STATUS_TASK_SCHEDULE).getValue()` no? If not set, will be `null`

Reply:
Whoops, yep